pymc/pytensorf.py

```diff
@@ -1000,16 +1001,16 @@ def reseed_rngs(
 
 
 def collect_default_updates(
-    inputs: Sequence[Variable],
     outputs: Sequence[Variable],
+    *,
+    inputs: Optional[Sequence[Variable]] = None,
```

Comment:
I guess inputs are optional precisely for the scan case? Is there any advantage/disadvantage to providing or not providing them (maybe it shortens the worst-case number of recursions on this function?)

Reply:
It has no performance reasons is just for user convenience so it doesn't have to pass inputs. You either pass all inputs or None and often users don't even realize what are the root inputs of their variables

Reply:
Plus it's a lot less verbose!